.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
```

Comment:
Does this field make sense as GitHub has a native assignee feature?

Reply:
good point, labels as well. This is something that came from github template